<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdHelpWindow</class>
 <widget class="QMainWindow" name="QxrdHelpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Help</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="m_HelpHomeButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="qxrdresources.qrc">
              <normaloff>:/images/go-home.png</normaloff>:/images/go-home.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_HelpBackButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="qxrdresources.qrc">
              <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_HelpForwardButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="qxrdresources.qrc">
              <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QxrdHelpBrowser" name="m_HelpBrowser">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="source">
           <url>
            <string>qrc:/help/qxrdhelpindex.html</string>
           </url>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
          <property name="openLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>To Do</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="m_ToDoList">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_FileMenu">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_EditMenu">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_WindowMenu">
    <property name="title">
     <string>W&amp;indow</string>
    </property>
    <addaction name="actionWindow"/>
   </widget>
   <addaction name="m_FileMenu"/>
   <addaction name="m_EditMenu"/>
   <addaction name="m_WindowMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWindow">
   <property name="text">
    <string>&amp;Window</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxrdHelpBrowser</class>
   <extends>QTextBrowser</extends>
   <header>qxrdhelpbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qxrdresources.qrc"/>
 </resources>
 <connections/>
</ui>
